docs/en/operation/Troubleshooting.md

```diff
@@ -70,11 +70,11 @@ After completing this setup, learn how [to attach](#to-attach).
 
 ### Debugging shell commands
 
-If you want to debug shell commands (e.g. `bin/alluxio fs ls /`), you can set the `ALLUXIO_USER_DEBUG_JAVA_OPTS` in
+If you want to debug shell commands (e.g. `bin/alluxio fs ls /`), you can set the `ALLUXIO_USER_ATTACH_OPTS` in
 `conf/alluxio-env.sh` as above:
 
 ```shell
-# Java 5 through 8
+# Java 8
 export ALLUXIO_USER_ATTACH_OPTS="-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=60000"
 # Java 9 and up
```

Comment:
should we just call this java 11 and not have users suspect we are compatible with java 9 and up